tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+   >
+ - using `-w` will create a working directory and populate it. It will copy over the references and the default `vpipe.config`, and create a handy `vpipe` short-cut script to invoke `snakemake`.
+
+   > **Tips**:
+   > To create and populate other new working directories, you can call `init_project.sh` from within the new directory :
+   ```bash
+   mkdir -p working_2
+   cd working_2
+   ../V-pipe/init_project.sh
+   ```
+
+## Running V-pipe
+
+
+Copy the `samples` directory you created in the step [Preparing a small dataset](#preparing-a-small-dataset) to this `working` directory. 
+You can display the directory structure with `tree samples` or `find samples`[^fancyfind].
+
+[^fancyfind]: `find samples | sed -e "s/[^-][^\/]*\// |/g" -e "s/|\([^ ]\)/|-\1/"` is overly long but produces slightly prettier output than `find samples`
+
+Check the parameters of the `vpipe.config` file with you favorite editor (`vim`, `emacs`, `nano`, [butterflies](https://xkcd.com/378/), etc.).
+For SNVs and Local (windowed) haplotype reconstruction, you will need at least the following options:
+
+*[SNVs]: Single Nucleotide Variants
+
+```
+[input]
+reference = references/NC_045512.2.fasta
+
+[output]
+snv = True
+local = True
+global = False
+
+[general]
+aligner = bwa
+```
+
+Check what will be executed:
+```bash
+./vpipe --dryrun -p --cores 2
+```
+
+As it is your first run of V-pipe, this will also generate the sample collection table.
+Check `samples.tsv` in your editor.
+
+Note that the demo files you downloaded have reads of length 150 only.
+V-pipe's default parameters are optimized for reads of length 250 ; add the third column in the file:
+
+```
+SRR10903401	20200102	150
+SRR10903402	20200102	150
+```
+
+> **Tips:** Always check the content of the `samples.tsv` file.
+>  - If you didn't use the [correct structure](#preparing-a-small-dataset), this file might end up empty or some entries might be missing.
+>  - You can safely delete it and re-run the `--dryrun` to regenerate it.
+
+Run the V-pipe analysis (the necessary dependencies will be downloaded and installed in conda environments managed by snakemake):
+
+```bash
+./vpipe -p --cores 2
+```
+
+## Output
+
+The Wiki contains an overview of the [output files](https://github.com/cbg-ethz/V-pipe/wiki/output).
+The output of the SNV calling is aggregated in a standard [VCF](https://en.wikipedia.org/wiki/Variant_Call_Format) file, located in
+`samples/`*\{hierarchy\}*`/variants/SNVs/snvs.vcf`, you can open it with your favorite VCF tools for aggregated or downstream processing.
+
+*[VCF]: Variant Call Format
+
+There is also a visual report generated in `samples/`*\{hierarchy\}*`/visualization/index.html` that you can simply open with your favorite web browser (Firefox, Chrome). You will get an output similar to that:
```

Comment:
This is not yet in the branch

Reply:
Monica and Kim's PR should actually hit your mailbox any time soon.

Reply:
But, it is my underatanding we won't release with this pre-release. It is not a good idea to rush the review process